eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Ratify.hs

```diff
@@ -395,14 +394,17 @@ updatePrevGovActionIdsChildren gas pgac =
 
 validCommitteeTerm ::
   ConwayEraPParams era =>
-  StrictMaybe (Committee era) ->
   PParams era ->
   EpochNo ->
+  GovAction era ->
```

Comment:
Not sure whether to mention the change of the signature in CHANGELOG, since it doesn't appear there to begin with (it's exported for testing). I suppose people could still use it, though.
If that's the case, maybe it's not worth changing the order of these parameters? I only did it so I can use the lambda case, but can't decide what's worth doing or not!

Reply:
`prevActionAsExpected, `withdrawalCanWithdraw` and `validCommitteTerm` all having `gasAction` as the first argument looks nice to be for consistency. 

I don't think lambda case is worth it :slightly_smiling_face: 

That being said, it will still change the type of the function, so mentioning it in the changelog is a good practice, but it will not be the end of the world if you don't do it.